spec/coder_spec.lua

```diff
@@ -1599,7 +1599,6 @@ describe("Pallene coder /", function()
         compile([[
             local z = 10
             export x = 2000
-            local x = 300
```

Comment:
Let's continue testing the situation where there is a deleted variable after the one that we are testing. This test cared more about the `global_id` than the name of the variables... While we are at it, we could also slightly improve the test by testing both at the middle and at the end of the list of variables.

```
local  a = 10
export b = 20
local  c = 30
export d = 40
```

```
assert(nil == test.a)
assert(20 == test.b)
assert(nil == test.c)
assert(40 == test.d)
```